src/Command/WorkerCommand.php

```diff
@@ -4,68 +4,44 @@
 
 namespace Baldinof\RoadRunnerBundle\Command;
 
-use Baldinof\RoadRunnerBundle\Worker\WorkerInterface;
-use Spiral\RoadRunner\Environment\Mode;
+use Baldinof\RoadRunnerBundle\Worker\WorkerResolverInterface;
+use Spiral\RoadRunner\EnvironmentInterface;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
-use Symfony\Component\Console\Style\SymfonyStyle;
 
 final class WorkerCommand extends Command
 {
     protected static $defaultName = 'baldinof:roadrunner:worker';
 
-    private WorkerInterface $worker;
+    private WorkerResolverInterface $workerResolver;
+    private EnvironmentInterface $environment;
 
-    public function __construct(WorkerInterface $worker)
+    public function __construct(
+        WorkerResolverInterface $workerResolver,
+        EnvironmentInterface $environment
+    )
     {
         parent::__construct();
 
-        $this->worker = $worker;
+        $this->workerResolver = $workerResolver;
+        $this->environment = $environment;
     }
 
     public function configure(): void
     {
         $this
-            ->setDescription('Run the roadrunner worker')
+            ->setDescription('Run the roadrunner')
             ->setHelp(<<<EOF
             This command should not be run manually but specified in a <info>.rr.yaml</info>
             configuration file.
             EOF);
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        if (getenv('RR_MODE') !== Mode::MODE_HTTP) {
```

Comment:
I think this message is still important if a user try to manually run the command.